### PR TITLE
Static OrdGetter with OrdTransfers []OrdTransfer

### PR DESCRIPTION
**Pull request by AugustNorden**

### CODE REVIEW

**Comment by AugustNorden** [comment]
[ordgetter.yml](https://github.com/RiemaLabs/modular-indexer-committee/pull/28/commits/f8fe0d4c0ebe209469880508a38989ad61ba4cf7) is too rough to be added for now

File: catchupStage_test.go [hunk 1]
```diff
@@ -9,19 +9,19 @@ import (
 	"github.com/RiemaLabs/modular-indexer-committee/ord/stateless"
 )
 
-func TestCatchUp(t *testing.T) {
+func TestCatchupStage(t *testing.T) {
```
Comment (archidoge0) [inline catchupStage_test.go:12]: Rename `TestCatchupStage` to `Test_CatchupStage`. 
Other tests shall be renamed like this.

File: .github/workflows/ordgetter.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: PostgreSQL-free OrdGetter Unit Test
```
Comment (archidoge0) [inline .github/workflows/ordgetter.yml:1]: Don't commit the .yaml at this commit.

PR one feature at once.

File: ord/getter/opiclienttest.go [hunk 3]
```diff
@@ -31,34 +79,15 @@ func (opi *OPIOrdGetterTest) GetBlockHash(blockHeight uint) (string, error) {
...
-		SELECT block_hash
-		FROM block_hashes
-		WHERE block_height = $1
-	`
-	err := opi.db.Raw(sql, blockHeight).Scan(&blockHash).Error
-	if err != nil {
-		return "", err
-	}
-	return blockHash, nil
+	return "", fmt.Errorf("block hash not found for block height: %d", blockHeight)
```
Comment (archidoge0) [inline ord/getter/opiclienttest.go:82]: Record the mapping from blockHeight to blockHash to another .csv file from pgAdmin.

File: ord/getter/opiclienttest.go [hunk 4]
```diff
@@ -31,34 +79,15 @@ func (opi *OPIOrdGetterTest) GetBlockHash(blockHeight uint) (string, error) {
...
-			AND onti.cursed_for_brc20 = false
-			AND oc."content" is not null AND oc."content"->>'p' = 'brc-20'
-		ORDER BY ot.id asc;
-		`
-	err := opi.db.Raw(sql, blockHeight).Scan(&ordTransfers).Error
-	if err != nil {
-		return make([]OrdTransfer, 0), err
+	var filteredOrdTransfers []OrdTransfer
+	for _, transfer := range opi.OrdTransfers {
+		if transfer.BlockHeight <= blockHeight {
```
Comment (archidoge0) [inline ord/getter/opiclienttest.go:88]: `transfer.BlockHeight <= blockHeight` is incorrect.

`transfer.BlockHeight == blockHeight`

**Pull request closed without merging**